examples/native/components/CSFExample/BooleanExample.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react-native';
+import { BooleanExampleButton } from './BooleanExampleButton';
+
+const BooleanExample: ComponentMeta<typeof BooleanExampleButton> = {
+  title: 'Boolean Example',
+  component: BooleanExampleButton,
+};
+
+export default BooleanExample;
+
+type BooleanExampleStory = ComponentStory<typeof BooleanExampleButton>;
```

Comment:
Just a minor thing, I think we can remove example from the name. Soon I will update all the others too.

Reply:
Sounds good, did that :+1: